.github/workflows/install-index-release.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         run: sbt 'runMain com.virtuslab.shared_indexes.Main project'
 
       - name: Check indexes
-        run: ls -lh workspace/cdn
+        run: find workspace/cdn -type f -exec du -h {} +
```

Comment:
Why not `ls -r`?

Reply:
Because it reverses the sorting order of ls :P I never knew there was a recursive flag for ls. It turns out to be `-R`. Nevertheless I won't be updating this again unless I will do other edits as it is not that important.